.circleci/config.yml

```diff
@@ -64,7 +64,21 @@ jobs:
       - run: CI=true npm run release:prep
       - run: TAG=$CIRCLE_TAG npm run release:validate
       - run: TAG=$CIRCLE_TAG npm run publish
-
+  containers_publish:
```

Comment:
Let's rename this to `publish_example_containers`

Reply:
done